src/test/java/com/facebook/openwifirrm/modules/ApiServerTest.java

```diff
@@ -559,4 +559,25 @@ void test_runRRM() throws Exception {
 		assertEquals(400, Unirest.put(url + "?mode=test123&venue=" + zone).asString().getStatus());
 		assertEquals(400, Unirest.put(url + "?venue=asdf&algorithm=" + algorithms.get(0)).asString().getStatus());
 	}
-}
+
+
+	@Test
+	@Order(2004)
+	void test_setSystem() throws Exception {
```

Comment:
nit: could you move this to just below `test_system()` and renumber the tests under it?